PowerEditor/src/Notepad_plus.rc

```diff
@@ -509,6 +509,7 @@ BEGIN
             MENUITEM "Word Completion",             IDM_EDIT_AUTOCOMPLETE_CURRENTFILE
             MENUITEM "Function Parameters Hint",    IDM_EDIT_FUNCCALLTIP
             MENUITEM "Path Completion",             IDM_EDIT_AUTOCOMPLETE_PATH
+            MENUITEM "Function Parameters Next",    IDM_EDIT_FUNCCALLTIP_NEXT

```

Comment:
1. I would rename it as "Function Parameters Next Hint"
2. Put it just blow `MENUITEM "Function Parameters Hint", IDM_EDIT_FUNCCALLTIP`
3. Add a new "Function Parameters Previous Hint" between "Function Parameters Hint" & "Function Parameters Next Hint"